core/src/components/input-password-toggle/input-password-toggle.ios.scss

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
I noticed that this and the corresponding `md` styles are copied directly from `input.md/ios.scss`. Would it make sense to update the selectors there instead to cover the new component, or maybe pull them into a shared mixin if we're trying to avoid component coupling? My worry is that we'll have to know to update the styles in both places if we want to change things while keeping the button appearance consistent.